essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-use-case-software-engineer.md

```diff
@@ -1,2 +1,102 @@
+<div align="center">
+  <p><small style="font-size: 70%"><em>Sample Appflowy Page</em></small></p>
+</div>
+
+
+
+## Benefits of using Appflowy as a Software Engineer
+Before we dive into the details, let's talk about the benefits of using Appflowy as a software engineer. First and foremost, Appflowy helps me stay organized. With its ability to create structured pages, I can keep all my projects, tasks, and notes in one place. Additionally, I can easily share my pages in markdown with my team and keep everyone on the same page.
+
+
+
+## Project Planning
+Now that we have our workspace set up, let's talk about how to use Appflowy for project planning. To plan a project in Appflowy, I create a new list for the project and add notes and tags as needed. For example, I might create a note for the project goal, a note for the requirements, and a note for each milestone. By tagging each note with the project name, I can easily find all the notes related to that project.
+
+![project plann](https://user-images.githubusercontent.com/89600478/235844017-2f00d682-5ddd-4fb5-a935-0055d0f3bb34.jpg)
+<div align="center">
+  <p><small style="font-size: 70%"><em>Project planning page</em></small></p>
+</div>
+
+## Development Tasks:
+Once I have my project plan in place, it's time to start working on the development tasks. To manage my development tasks, I create a new list for each task and add notes and tags as needed. For example, I might create a note for the task description, a note for any subtasks, and a note for the estimated time to complete. By tagging each note with the project name and task name, I can easily find all the notes related to that task.
+
+
+<div align="center">
+  <img src="https://user-images.githubusercontent.com/89600478/235844308-d90c9446-732f-4b0a-8808-f7ace7f3e774.png">
+</div>
+
+
+<div align="center">
+  <p><small style="font-size: 70%"><em>Yes I use boards to design my Database schema</em></small></p>
+</div>
+
+ 
+## Bugs and Issues
+Of course, no software project is without its bugs and issues. That's where Appflowy's bug tracking features come in. To track bugs and issues, I create a new list for each issue and add notes and tags as needed. For example, I might create a note for the bug description, a note for the steps to reproduce, and a note for the priority level. By tagging each note with the project name and issue name, I can easily find all the notes related to that issue.
+
+
+
+
+## Project Completion
+Once all the development tasks are completed and all the bugs are fixed, it's time to wrap up the project. To manage the project completion, I create a new list for the project and add notes and tags as needed. For example, I might create a note for the final deliverables, a note for any outstanding issues, and a note for the project status. By tagging each note with the project name, I can easily find all the notes related to that project.
+
```

Comment:
Same here, add some screenshots about "create a note for the final deliverables, a note for any outstanding issues, and a note for the project status." will help the reader understand what you mean.  I'm a bit confused about what a complete project management setup would look like in Appflowy.
You talked about creating a new list in every step (Project Planning, Development Tasks, Bugs and Issues, Project Completion), could you show some screenshots of them? like a overview of a whole project

Reply:
@AbubakrChan 
Same here, it needs some screenshots for your example.